package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "cache-wrapper",
+  "description": "System aggregator and data logic layer",
+  "version": "0.1.0",
+  "homepage": "https://github.com/holidayextras/cache-wrapper",
+  "author": {
+    "name": "Shortbreaks",
+    "email": "shortbreaksdev@holidayextras.com"
+  },
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:holidayextras/cache-wrapper.git"
+  },
+  "license": "MIT",
+  "main": "index",
+  "engines": {
+    "node": ">= 0.10.0"
+  },
+  "scripts": {
+    "start": "node index.js",
+    "test": "grunt jshint && grunt jscs && istanbul cover _mocha -- test/**/*Test.js -R spec"
+  },
+  "dependencies": {
+    "catbox": "4.2.0",
+    "catbox-memory": "1.1.1",
+    "catbox-redis": "1.0.4",
+    "lodash": "3.10.1",
+    "q": "1.4.1"
+  },
+  "devDependencies": {
+    "chai": "2.2.0",
+    "chai-as-promised": "5.1.0",
+    "grunt": "0.4.4",
+    "grunt-contrib-jshint": "0.10.0",
+    "grunt-jscs": "0.8.1",
+    "istanbul": "0.3.7",
+    "make-up": "1.0.4",
```

Comment:
https://github.com/holidayextras/plugin-http/compare/makeup?expand=1 is another that has the work but unmerged so you can see the diff